packages/sources/kaiko/src/transport/trades.ts

```diff
@@ -21,10 +21,12 @@ interface ResponseSchema {
     request_time: string
     instruments: string[]
     start_timestamp: number
+    end_timestamp: number
+    extrapolate_missing_values: string
```

Comment:
In the `fixtures.ts` this is `false` - could this actually be a `boolean`?

Reply:
good catch, my bad.
Though this isn't currently used (along with 90% of the other params), so arguably doesn't need to be added.